### DISCUSSION TITLE
Issues滿載~

### DISCUSSION THREAD

**Discussion by bestian** [3dw/auto20-next#224] [Announcements]
@3dw/educator 

Open Issues滿載啦
上限是10個Open Issues。接下來請先solve Issue再提新的Issue, 謝謝~

**Comment by leechiuhui** [comment]
收到，老師。
關於 issue #220 需要 「double check完再close this issue比較保險」 的這個部分，
我明天10月9日上午，精神較好時來處理喔～🫡
